index.js

```diff
@@ -26,7 +31,7 @@ app.get('/api/reviewcount/:gameId', (req, res) => {
 
 app.get('/api/reviewcount/recent/:gameId', (req, res) => {
   const sqlText = 'SELECT SUM(positive) as pos, SUM(negative) as neg '
-                  + 'FROM reviews_graph WHERE date >= CURDATE()-30 AND date <= CURDATE() '
+                  + 'FROM reviews_graph WHERE date >= DATE_SUB(CURDATE(),INTERVAL 30 DAY)  '
```

Comment:
Looks like a powerful use of raw SQL. Nice!

Reply:
Thanks!